src/main/java/core/basesyntax/vehicles/Truck.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.vehicles;
+
+import core.basesyntax.Machine;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck started working");
```

Comment:
The message 'Truck started working' is not very informative. It would be better to indicate what the Truck is doing specifically, for example, 'Truck is transporting goods'.